modules/auxiliary/scanner/winrm/winrm_cmd.rb

```diff
@@ -28,15 +28,6 @@ def initialize
         OptString.new('USERNAME', [ true, 'The username to authenticate as'])
       ]
     )
-
```

Comment:
Removed this because it's redundant with the `include Msf::Exploit::Remote::WinRM` on L10.